omnibus/config/projects/agent.rb

```diff
@@ -110,32 +109,23 @@
   dependency 'curl'
 end
 
-dependency 'cacerts'
 # creates required build directories
 dependency 'datadog-agent-prepare'
 
 # Datadog agent
 dependency 'datadog-agent'
 
 # Additional software
-dependency 'pip'
-dependency 'datadog-agent-integrations'
+dependency 'cacerts'
 dependency 'datadog-a7'
-dependency 'datadog-agent-env-check'
```

Comment:
in the effort of reducing the number of defs hence the build time, I've moved the contents of this def into `datadog-agent-integrations`